versions.json

```diff
@@ -306,6 +322,22 @@
         ],
         "fpm_directories": ["usr/bin"]
     },
+    "nanodbc": {
+        "commitish": "aaa3b89db6e5852756a9b6fdfdce404af73adfb6",
```

Comment:
interesting there hasn't been a release since 2016... ?

Reply:
right.  i went with master since the tags were saying master builds and passes the tests.  I think they also mention it is ok to use the tip if it passes tests.

Reply:
agreed.   just surprising.    carry on.